### DISCUSSION TITLE
401 Unauthorized On Admin Panel Login! :(

### DISCUSSION THREAD

**Discussion by ArnavK-09** [medusajs/medusa#6053] [Q&A] [answered]
## Tried all these admin cors

- "/github\\.dev$/"
- ADMIN_CORS=/http:\/\/*/
- /\.app\.github\.dev/
- "*"
- /http:\/\/*/
- /https:\/\/*/

# Devloping on Github Code Spaces

![image](https://github.com/medusajs/medusa/assets/69188140/e0cf380e-a6a2-4bab-8e04-80528d254ac5)
![image](https://github.com/medusajs/medusa/assets/69188140/3065a2d9-ae84-4931-a525-1e37bfd69058)
![image](https://github.com/medusajs/medusa/assets/69188140/5471fc6d-226f-4350-9482-ccdb3455275a)
![image](https://github.com/medusajs/medusa/assets/69188140/c4c7c6e8-d5d6-4da1-8c12-2d8a32b2c237)

**Comment by alightinastorm** [comment]
well it does say credentials don't match?

**Comment by ghost** [comment]
I am facing the same issue. I am sure I am passing the correct credentials but it keeps throwing the same error @ArnavK-09 has.

**Reply by ArnavK-09** [reply, marked as answer]
Try putting ADMIN_CORS same as your admin url (don't include / at end) this worked for me
+ 
Push admin creds to db
